pylti/tests/test_flask.py

```diff
@@ -173,9 +182,11 @@ def test_access_to_oauth_resource(self):
         self.assertFalse(self.has_exception())
 
     def test_access_to_oauth_resource_name_passed(self):
+        # pylint: disable=invalid-name
         """
         Check that name is returned if passed via initial request
         """
+        # pylint: disable=maybe-no-member
```

Comment:
@carsongee  fix this one , I made response iterate-able by using yield. Otherwise i was not able to access dict of response because it was not iterate-able. No pyltint is not giving maybe-no-member messages